src/pages/Login.js

```
@@ -1,85 +1,97 @@
-                            className="px-[3vw] py-[2vw] text-[4vw] my-[6vw] bg-[#272727] rounded-3xl text-[#CFFF0F]  focus:border-[#CFFF0F]"
-                            required
-                        />
+  return (
+    <div className="w-full h-full flex flex-col mt-[14vw] items-center justify-start decoration-none">
+      <div className="h-[10vw] w-[20vw] rounded-3xl bg-[#272727] flex justify-center items-center text-[4vw]">
+        {transportMode}
+      </div>
+      {otpSent ? (
+        <div className="p-[20vw] mt-[20vw]">
+          <h1 className="text-[7vw]">Enter OTP</h1>
+          <></>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)